{{ cookiecutter.package_name }}/.rtd-environment.yml

```diff
@@ -0,0 +1,8 @@
+name: rtd_sunpy
+channels:
+  - conda-forge
+dependencies:
+  - python=3.10
```

Comment:
```suggestion
  - python=3.12
```
Time to update this?

Reply:
Given that the min python requirement is 3.10, is it not better to have the env run off that so we know it works for users?

Reply:
Yeah that's a good point.